pkg/instance/instance.go

```diff
@@ -47,6 +49,13 @@ const (
 // reset token.
 var passwordResetValidityDuration = 15 * time.Minute
 
+var twoFactorTOTPOptions = totp.ValidateOpts{
+	Period:    30, // 30s
+	Skew:      4,  // 30s +- 4*30s = [-2min; 2,5min]
```

Comment:
I think we should have a larger window. Mail can be slow, particularly with greylisting. Maybe something 5 minutes